src/patternfly/components/Alert/alert.scss

```diff
@@ -34,9 +34,9 @@
   --#{$alert}__icon--FontSize: var(--pf-t--global--icon--size--md);
 
   // Title
-  --#{$alert}__title--FontFamily: var(--pf-t--global--font--family--heading);
-  --#{$alert}__title--FontSize: var(--pf-t--global--font--size--heading--xs);
-  --#{$alert}__title--FontWeight: var(--pf-t--global--font--weight--heading);
+  --#{$alert}__title--FontFamily: var(--pf-t--global--font--family--body);
+  --#{$alert}__title--FontSize: var(--pf-t--global--font--size--body--default);
```

Comment:
It's fine to have these as component vars, especially if we want to open the font-family and font-size to be themed, but technically we don't need them, since they should inherit both of these styles from the global styles. In particular, I'd say we drop `--#{$alert}__title--FontFamily` unless you want users to customize it. We only set the font-family to `--pf-t--global--font--size--body--default` in 2 other places:
* Multiple file upload "separator" text (the word "or" here - https://patternfly-pr-6775.surge.sh/components/file-upload/multiple-file-upload#basic) https://github.com/patternfly/patternfly/blob/625dcc9e49a0baf1867f40eb7c02e7c0adb93b05/src/patternfly/components/MultipleFileUpload/multiple-file-upload.scss#L44
  * That's because the word "or" is part of the title, and the title uses the `--heading` font-family, so we need to reset it for the "or" text.
* Popover title text, which I'd advocate removing unless we expect the font-family to be themed. My guess as to why it's set is similar to what I'm assuming the change in this PR is. There was a declaration for `font-family` to set it to the `--heading` font, so we updated it to use `--body` when that isn't necessary since it will be `--body` by default from PF's global styles